src/storage/src/storage_state.rs

```diff
@@ -414,11 +414,14 @@ impl<'w, A: Allocate> Worker<'w, A> {
             return;
         }
 
-        let mut disconnected = false;
-
-        let mut last_stats_time: Option<Instant> = None;
+        // The last time we reported statistics.
+        let mut last_stats_time = Instant::now();
```

Comment:
This is a slight change to the previous behavior. I figured it's probably not important that we collect statistics immediately after the first worker step. If that's wrong I can change it back into an `Option`, just makes the code a bit more noisy.